test/wasm/test4.wast

```diff
@@ -9,11 +9,11 @@
 	(func $b (param i32) (result f32)
 	  (return
             (if_else (i32.const 0) (f32.const 1)
-                   (f32.const 2))))
+                     (f32.const 2))))
 	(func $a (param i32) (result f32)
 	  (block (call $b (i32.const 1)) (call $c)))
-   (func $e (param i32) (result i32)
-         (i32.const 0))
+        (func $e (param i32) (result i32)
+          (i32.const 0))
```

Comment:
Looks like these two lines are indented one space further than you were planning.

```
        (func $e (param i32) (result i32)
          (i32.const 0))
```